### ISSUE TITLE
Update datasets to use "latest" timestamp

### ISSUE THREAD

**Issue by edelarua** [insightsengineering/tlg-catalog#25]
After generating new `rcd_2023_03_17` datasets, tlg-catalog should be updated to use newest version of data. To avoid having to make this change in the future (and since we are able to track template changes caused by data updates via `scda.test`), we should update to use "latest" instead of  "rcd_2023_03_17".

**Comment by shajoezhu** [comment]
Thanks a lot @edelarua for raising this. and thanks for blocking as well. Let me think a little bit more about this.

**Comment by shajoezhu** [comment]
I agree with this. I think given rcd has been through a lot of refactoring, a lot of these templates will be changed. However, at the same time we are also changing other upstream packages, if we overwrite all these tempalte snapshot at the same time, there is a chance (can't say it is a big chance, but there is a risk) that we miss catching these changes. I would suggest to update these in batches.
